### DISCUSSION TITLE
Quick Book Integration

### DISCUSSION THREAD

**Discussion by sreenak** [ruckus/quickbooks-ruby#559] [Q&A] [answered]

**Comment by ruckus** [comment, marked as answer]
Hi @sreenak, I cannot really speak to that tutorial per se. I think it might be out-dated (?). It's not necessarily an official tutorial either.

Your error boils down to the `$qb_oauth_consumer` object being nil and you're attempting to invoke a method on it. 

You will need to create your own consumer object. Please refer to the first part of the README:

https://github.com/ruckus/quickbooks-ruby#getting-started--initiating-authentication-flow-with-intuit

**Reply by sreenak** [reply]
@ruckus  Thanks for the reply .Modified to new version. Now working.
